custom_components/openhasp/__init__.py

```diff
@@ -504,6 +518,14 @@ async def async_change_page(self, page):
         )
         self.async_write_ha_state()
 
+    async def async_command_wrapper(self, command, keyword, parameters):
+        """Sends commands directly to the plate entity (as a wrapper for MQTT commands sent to hasp/<nodename>/command)"""
+        cmd_topic = f"{self._topic}/" + command
+
+        self.hass.components.mqtt.async_publish(
+            cmd_topic, (keyword + " " + parameters).strip(), qos=0, retain=False
```

Comment:
```suggestion
            cmd_topic, f"{keyword} {parameters.strip()}", qos=0, retain=False
```

Reply:
We need to strip the result of `(keyword + " " + parameters)` to ensure that no space remains even if parameters is empty.
`(f"{keyword} {parameters}").strip()` is possible?

Reply:
didn't realize you expected leading whitespaces in keyword

Yes that works, even without the ()